README.md

```diff
@@ -16,8 +16,11 @@
 [jenkins-shield]: https://img.shields.io/badge/Download-Jenkins-purple.svg
 [license-shield]: https://img.shields.io/badge/License-Apache%202.0-white.svg
 [migration-shield]: https://img.shields.io/badge/Wiki-Migrating%20from%20V3-darkgreen.svg
+[GatewayIntent]: https://docs.jda.wiki/net/dv8tion/jda/api/requests/GatewayIntent.html
+[JDABuilder]: https://docs.jda.wiki/net/dv8tion/jda/api/JDABuilder.html
+[DefaultShardManagerBuilder]: https://docs.jda.wiki/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.html
 
-<img align="right" src="https://github.com/discord-jda/JDA/blob/assets/assets/readme/logo.png?raw=true" height="200" width="200">
+<img align="right" src="https://github.com/discord-jda/JDA/blob/assets/assets/readme/logo.png?raw=true" height="150" width="150">
 
 [ ![maven-central][] ][download]
```

Comment:
```suggestion
[![maven-central][]][download]
```

Apply to all other shields, this removes the link overflow 
![image](https://github.com/discord-jda/JDA/assets/41875020/695a94c1-ec86-4b90-97d2-9d68a2bccad0)